README.md

```
@@ -221,31 +229,118 @@ tally check --max-lines 100 --skip-blank-lines --skip-comments Dockerfile
 
+tally supports multiple output formats for different use cases.
+
 ### Text (default)
 
+Human-readable output with colors and source code snippets:
+
+```bash
+tally check Dockerfile
+```
+
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)